src/components/SingleArticle.jsx

```diff
@@ -30,12 +42,18 @@ export const SingleArticle = () => {
   } else {
     return (
       <>
-        <img src={articleImg} />
-        <p>{articleTitle}</p>
+        <h2>{articleTitle}</h2>
         <p>
           By {articleAuthor}, publishing date: {articleDate.slice(0, 10)}
         </p>
-        <p>Votes: {articleVotes}</p>
+        <img className="imgArt" src={articleImg} />
+
+        <p>
+          <button onClick={upVote}>🔺</button>
+          &nbsp;&nbsp;Votes: {articleVotes}&nbsp;&nbsp;
+          <button onClick={dnVote}>🔻</button>
+        </p>
+
         <p>{articleBody}</p>
         <Comments id={id} />
       </>
```

Comment:
Optionally you could also add optimistic rendering here and handle the error if you get one. You cna test this by passing through an incorrect body and handle the error with conditional rendering.